packages/airnode-node/src/api/unsafe-evaluate.ts

```diff
@@ -0,0 +1,16 @@
+import { ethers } from 'ethers';
+
+/**
+ * This function is dangerous. Make sure to use it only with Trusted code.
+ */
+export const unsafeEvaluate = (input: any, code: string) => {
+  return Function(`
+    "use strict";
+    const [ethers, input] = arguments
+    const {BigNumber} = ethers;
```

Comment:
@aquarat shall we have this line here? I am not sure why only expose BigNumber drectly and not other stuff.

Reply:
We can remove it, no problem :)

Reply:
...that was fast @ commit :P